cmd/juju/resource/upload.go

```diff
@@ -155,5 +156,8 @@ func (c *UploadCommand) upload(rf resourceValue, client UploadClient) error {
 	}
 	defer f.Close()
 	err = client.Upload(rf.application, rf.name, rf.value, f)
-	return errors.Trace(err)
+	if err := block.ProcessBlockedError(err, block.BlockChange); err != nil {
```

Comment:
Looking at it now, it could have been just 
```
return block.ProcessBlockedError(err, block.BlockChange)
```

Reply:
Yeah totally, probably not worth a rebuild since we want to get this into 2.6.4.